Assets/MRTK/Tools/BuildCommon/AssemblyDefinition.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.MixedReality.Toolkit.Tools.Build
+{
+    /// <summary>
+    /// A representation of a Unity assembly definition file.
+    /// The structure of this data is defined by the JSON file format here:
+    /// https://docs.unity3d.com/Manual/ScriptCompilationAssemblyDefinitionFiles.html
+    /// </summary>
+    public struct AssemblyDefinition
```

Comment:
We have https://github.com/microsoft/MixedRealityToolkit-Unity/blob/mrtk_development/Assets/MRTK/Core/Utilities/Editor/AssemblyDefinition.cs already, which we use for reading from json etc. Are you able to reuse it here?

Reply:
:S

Oops face